examples/link/link.html

```diff
@@ -113,7 +113,7 @@ <h2 id="kbd_label">Keyboard Support</h2>
         </tr>
       </thead>
       <tbody>
-        <tr>
+        <tr data-test-id="key-enter">
```

Comment:
I like that you're using the tables found in the examples to inform the tests. We'll want to keep a close eye on whether that's sufficient.